compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -1497,6 +1497,48 @@ static LogicalResult setRootConfig(mlir::FunctionOpInterface entryPointFn,
       entryPointFn, fftOp, tileSizes, DispatchLoweringPassPipeline::CPUDefault);
 }
 
+/// Sets the lowering configuration for dispatch region for
+/// linalg_ext.winograd.input_transform root op.
+static LogicalResult
+setRootConfig(mlir::FunctionOpInterface entryPointFn,
+              IREE::LinalgExt::WinogradInputTransformOp inputOp) {
+  assert(!getLoweringConfig(inputOp) && "expected lowering_config is not set");
+  auto outShape = inputOp.getOutputOperandType().getShape();
+  SmallVector<int64_t> distTileSizes(outShape.begin() + 2, outShape.end());
+  distTileSizes = llvm::map_to_vector(distTileSizes, [](int64_t size) {
+    return std::min<int64_t>(size, clDefaultDistTileSize);
+  });
+  distTileSizes[1] = distTileSizes[2] = 0;
+  TileSizesListType tileSizes;
+  tileSizes.push_back(distTileSizes);
+  SmallVector<int64_t> vecTileSizes(4, 1);
+  tileSizes.push_back(vecTileSizes);
+  return setOpConfigAndEntryPointFnTranslation(
+      entryPointFn, inputOp, tileSizes,
+      DispatchLoweringPassPipeline::CPULinalgExtTileAndVectorize);
+}
+
+/// Sets the lowering configuration for dispatch region for
+/// linalg_ext.winograd.input_transform root op.
+static LogicalResult
+setRootConfig(mlir::FunctionOpInterface entryPointFn,
+              IREE::LinalgExt::WinogradOutputTransformOp outputOp) {
+  assert(!getLoweringConfig(outputOp) && "expected lowering_config is not set");
+  auto inShape = outputOp.getInputOperandType().getShape();
+  SmallVector<int64_t> distTileSizes(inShape.begin() + 2, inShape.end());
+  distTileSizes = llvm::map_to_vector(distTileSizes, [](int64_t size) {
+    return std::min<int64_t>(size, clDefaultDistTileSize);
+  });
+  distTileSizes[1] = distTileSizes[2] = 0;
+  TileSizesListType tileSizes;
+  tileSizes.push_back(distTileSizes);
+  SmallVector<int64_t> vecTileSizes(4, 1);
```

Comment:
Where is the magic number `4` come from? Can we use `getVectorSize` method here?

https://github.com/openxla/iree/blob/56541e4a9a2b34fab011f67a4e380031904dc59f/compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp#L282-L289